cgmes/cgmes-model/src/main/resources/CIM16.sparql

```diff
@@ -846,7 +846,7 @@ WHERE {
         cim:StaticVarCompensator.inductiveRating ?inductiveRating ;
         cim:StaticVarCompensator.slope ?slope ;
         cim:StaticVarCompensator.sVCControlMode ?controlMode ;
-        cim:StaticVarCompensator.voltageSetPoint ?voltageSetPoint .
+        cim:StaticVarCompensator.voltageSetpoint ?voltageSetpoint .
```

Comment:
Change to:
`cim:StaticVarCompensator.voltageSetPoint ?voltageSetpoint`
The attribute name in files should not be changed